src/utils/Intercom.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { useRouteMatch } from 'react-router'
+import { useVariation, Variation } from 'utils/hooks/useVariation'
+
+export const Intercom: React.FC = () => {
+  const variation = useVariation()
+  const isIntercomMatch = useRouteMatch(
+    '/:locale(no-en|no)/new-member/:place(offer|sign|download)',
+  )
+  React.useEffect(() => {
+    if ([Variation.IOS, Variation.ANDROID].includes(variation!)) {
+      return
+    }
+
+    const elementById = window.document.getElementById('intercom-script')
+    if (isIntercomMatch && !elementById) {
+      const script = `
+      window.intercomSettings = {
+        app_id: "ziqa7goa"
+      };
+
+      (function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',w.intercomSettings);}else{var d=document;var i=function(){i.c(arguments);};i.q=[];i.c=function(args){i.q.push(args);};w.Intercom=i;var l=function(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/ziqa7goa';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);};if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})();
+    `
+      const scriptTag = window.document.createElement('script')
+      scriptTag.id = 'intercom-script'
+      scriptTag.nonce = window.document.querySelector<HTMLScriptElement>(
+        '[nonce]',
+      )?.nonce
+      scriptTag.innerHTML = script
+      window.document.body.append(scriptTag)
+    } else if (
+      isIntercomMatch &&
+      typeof (window as any).Intercom !== 'undefined'
+    ) {
+      ;(window as any).Intercom('boot')
+    } else if (
+      !isIntercomMatch &&
+      typeof (window as any).Intercom !== 'undefined'
+    ) {
+      ;(window as any).Intercom('shutdown')
+    }
```

Comment:
Not sure what's going on here haha, but I trust you do

Reply:
Basically turning it on if we should display it and off if we shouldn't maybe that should be clarified though?